packages/common/src/spokenForms/marks.ts

```diff
@@ -0,0 +1,73 @@
+import { PartialMark } from "..";
+import { HatColor, HatShape } from "../types/command/hatStyles.types";
+
+const hatColors: Record<HatColor, string | null> = {
+  blue: "blue",
+  green: "green",
+  red: "red",
+  pink: "pink",
+  yellow: "yellow",
+  userColor1: "navy",
+  userColor2: "apricot",
+
+  default: null,
```

Comment:
This is a bit different from the other situations where we use a new value, right? In those situations, it means that there is no spoken form. So we'll raise a no spoken form error. Whereas in this situation, it just means that the spoken form is omitted. I wonder if we just want to map this to an empty string? I don't have a strong feeling either way.

Reply:
nvm I think this is consistent